app/views/errors/notFound.scala.html

```diff
@@ -0,0 +1,15 @@
+@import db.ModelService
+@import db.impl.access.UserBase
+@import ore.{OreConfig, OreEnv}
+@()(implicit messages: Messages, session: Session, requestHeader: RequestHeader, request: Request[_] = null,
+        service: ModelService, config: OreConfig, users: UserBase, env: OreEnv)
+
+@bootstrap.layout("404 Not Found", user = false) {
```

Comment:
Please add all constant strings to the `messages` file and access through `Messages`.